ducktape/services/service.py

```diff
@@ -52,21 +79,71 @@ def start(self):
                     "Service: %s, node.account: %s" % (self.__class__.__name__, str(node.account)))
             node.account.set_logger(self.logger)
 
-            self.logger.debug("Forcibly cleaning node %d on %s", idx, node.account.hostname)
-            self._clean_node(node)
+        self.allocated = True
+
+    def start(self):
+        """Start the service on all nodes."""
+        self.logger.info("Starting service: %s" % self.__class__.__name__)
+
+        if not self.allocated:
+            self.allocate_nodes()
+
+        self.logger.debug("Killing processes and attempting to clean up before starting nodes.")
+        for node in self.nodes:
+            # Added precaution - kill running processes
+            self._kill_running_processes(node)
```

Comment:
This was the call I was talking about getting rid of here, or moving to a utility. Its implementation just kills all java processes.

This is especially important if we make the LocalhostCluster work :)